x-pack/legacy/plugins/siem/public/components/events_viewer/index.tsx

```diff
@@ -131,31 +152,30 @@ const StatefulEventsViewerComponent = React.memo<Props>(
     const handleOnMouseLeave = useCallback(() => setShowInspect(false), []);
 
     return (
-      <WithSource sourceId="default">
-        {({ indexPattern, browserFields }) => (
-          <div onMouseEnter={handleOnMouseEnter} onMouseLeave={handleOnMouseLeave}>
-            <EventsViewer
-              browserFields={browserFields}
-              columns={columns}
-              id={id}
-              dataProviders={dataProviders!}
-              end={end}
-              filters={filters}
-              indexPattern={indexPattern}
-              isLive={isLive}
-              itemsPerPage={itemsPerPage!}
-              itemsPerPageOptions={itemsPerPageOptions!}
-              kqlMode={kqlMode}
-              onChangeItemsPerPage={onChangeItemsPerPage}
-              query={query}
-              showInspect={showInspect}
-              start={start}
-              sort={sort!}
-              toggleColumn={toggleColumn}
-            />
-          </div>
-        )}
-      </WithSource>
+      <div onMouseEnter={handleOnMouseEnter} onMouseLeave={handleOnMouseLeave}>
+        <EventsViewer
+          browserFields={browserFields ?? {}}
+          columns={defaultModel.columns}
+          id={id}
+          dataProviders={dataProviders!}
+          end={end}
+          filters={{ ...filters, ...defaultFilters }}
+          headerFilterGroup={headerFilterGroup}
+          indexPattern={indexPatterns ?? { fields: [], title: '' }}
```

Comment:
Passing empty defaults for both `browserFields` and `indexPattern` to retain the default loading behavior that is present when using the `WithSource` HoC. Will look into creating a better initial loading flow in `Part II`.